frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -126,5 +122,6 @@ export const InventoryItemStatus: React.FC<InventoryItemStatusProps> = require('
   .InventoryItemStatus;
 export const InventoryItemLoading: React.FC = require('@console/shared/src/components/dashboard/inventory-card/InventoryCard')
   .InventoryItemLoading;
+export const ClusterOverviewDetailsItem: React.FC<ClusterOverviewDetailsItemProps> = require('@console/shared/src/components/dashboard/details-card/DetailItem').default;
```

Comment:
I think that the component is meant to be used for all Overview pages 

```suggestion
export const OverviewDetailsItem: React.FC<ClusterOverviewDetailsItemProps> = require('@console/shared/src/components/dashboard/details-card/DetailItem').default;
```